llvm/test/MC/ARM/pcrel-ldr-relocs.s

```diff
@@ -0,0 +1,45 @@
+@ RUN: llvm-mc -filetype=obj -triple=armv7 %s -o %t
+@ RUN: llvm-readelf -r %t | FileCheck %s --check-prefix=ARM
+@ RUN: llvm-objdump -d --triple=armv7 %t | FileCheck %s --check-prefix=ARM_ADDEND
+@ RUN: llvm-mc -filetype=obj -triple=thumbv7 %s -o %t
+@ RUN: llvm-readelf -r %t | FileCheck %s --check-prefix=THUMB
+@ RUN: llvm-objdump -d --triple=thumbv7 %t | FileCheck %s --check-prefix=THUMB_ADDEND
+
+@ ARM: R_ARM_LDR_PC_G0
+@ ARM: foo1
```

Comment:
What's the text you're checking for here? Again, just being paranoid that:
```
R_ARM_LDR_PC_G0
random_symbol
R_SOMETHING_ELSE
foo1
```

Would match this too.

If they are on the next line you can do ARM-NEXT: and lit will make sure there's no extra stuff there, or regex is an option if you've got stuff in between that varies e.g. offsets and the like.

Reply:
https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-next-directive

And there's one for same line, I think. They should all be listed in that doc somewhere.